lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -229,7 +229,11 @@ def always_ignore_tables
               nil
             end
 
-          ['schema_info', 'schema_migrations', 'ar_internal_metadata', sessions_table].compact
+          [
```

Comment:
Didn't we say at standup that we'd keep `schema_info` in the list too, just in case? I don't think the odds are high that anyone would have a table of that name that they wanted `declare_schema` to manage, so I don't think it's very risky to leave it in the list for now, until we make all of these configurable.

Reply:
My bad, I thought we decided we could remove it. I'll add it back.